### DISCUSSION TITLE
Board UI does not work in Wrapped Mode

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#249] [Bug Reports]
This is a 'known' bug from when the Board repo was refactored. Since Wrapped isn't supported in the Web UI support for it was left off the Board repo.

But the Rules CLI can still create Wrapped games, so would be good to get the board fixed

Creating this as a tracking ticket!

**Comment by coreyja** [comment]
This 'issue' is that we are currently using an SVG `polyline` to render the snake

And when we have points that aren't near each other the Polyline connects them.

One possible solution would be to split the snake into multiple polylines potentially?

Source: https://github.com/BattlesnakeOfficial/board/blob/main/src/lib/components/SvgSnakeBody.svelte#L194

**Comment by und3f** [comment]
Created PR for displaying wrapped snake on the board https://github.com/BattlesnakeOfficial/board/pull/77

**Reply by coreyja** [reply]
That PR was merged and deployed!

Thanks @und3f for the contribution and getting this fixed!

**Comment by coreyja** [comment]
Implemented and Deployed!

Thanks for the implementation in https://github.com/BattlesnakeOfficial/board/pull/77 @und3f
